crashtracker-ffi/src/crash_info/mod.rs

```diff
@@ -86,16 +83,16 @@ pub unsafe extern "C" fn ddog_crashinfo_add_counter(
 /// call.
 #[no_mangle]
 #[must_use]
-pub unsafe extern "C" fn ddog_crashinfo_add_file(
+pub unsafe extern "C" fn ddog_crasht_CrashInfo_add_file(
     crashinfo: *mut CrashInfo,
-    name: CharSlice,
-) -> CrashtrackerResult {
+    filename: CharSlice,
+) -> Result {
     (|| {
         let crashinfo = crashinfo_ptr_to_inner(crashinfo)?;
-        let name = name.to_utf8_lossy();
-        crashinfo.add_file(&name)
+        let filename = filename.to_utf8_lossy();
+        crashinfo.add_file(&filename)
```

Comment:
Really unrelated to this PR, but I noticed that `CrashInfo::add_file` seems to read the entire file to memory before trying to do anything with its contents.

Perhaps it may be worth adding a hard limit there, in case we get fed the wrong thing? (Insert crowdstrike broken file joke/reference?)